lighthouse/routes/v3/beckman_v3_routes.py

```diff
@@ -0,0 +1,48 @@
+from flask import Blueprint, request
+from flask_cors import CORS
+
+from lighthouse.types import FlaskResponse
+
+from lighthouse.constants.fields import FIELD_EVENT_TYPE
+from lighthouse.classes.beckman_v3 import Beckman
+
+from lighthouse.hooks.beckman_events import create_plate_event
+from lighthouse.routes.common.beckman import get_failure_types, get_robots
+from lighthouse.routes.common.cherrypicked_plates import create_plate_from_barcode, fail_plate_from_barcode
+from lighthouse.routes.common.plate_events import create_plate_event as v1_create_plate_event
+
+bp = Blueprint("beckman_v3_routes", __name__)
+CORS(bp)
+
+
+@bp.get("/beckman/robots")
+def get_robots_endpoint() -> FlaskResponse:
+    return get_robots()
+
+
+@bp.get("/beckman/failure-types")
+def get_failure_types_endpoint() -> FlaskResponse:
+    return get_failure_types()
+
+
+@bp.get("/plate-events/create")
+def create_plate_event_endpoint() -> FlaskResponse:
+    event_type = request.args.get(FIELD_EVENT_TYPE, type=str)
+    if event_type in [
+        Beckman.EVENT_SOURCE_UNRECOGNISED,
+        Beckman.EVENT_SOURCE_COMPLETED,
+        Beckman.EVENT_SOURCE_ALL_NEGATIVES,
+        Beckman.EVENT_SOURCE_NO_PLATE_MAP_DATA,
+    ]:
+        return create_plate_event()
+    return v1_create_plate_event()
+
+
+@bp.get("/cherrypicked-plates/create")
+def create_plate_from_barcode_endpoint() -> FlaskResponse:
+    return create_plate_from_barcode()
+
+
+@bp.get("/cherrypicked-plates/fail")
+def fail_plate_from_barcode_endpoint() -> FlaskResponse:
+    return fail_plate_from_barcode()
```

Comment:
There's a lot of repetition in this file compared with the v1 file.  I would think as a v3, where v1 is also defined in the project, if the functionality hasn't changed between v1 and v3, you'd pass the request through the earlier interface.  i.e. you'd use something like `from lighthouse.routes.v1.beckman_routes import fail_plate_from_barcode_endpoint as v1_fail_plate_from_barcode` then the implementation here would be calling `return v1_fail_plate_from_barcode()`.  This way it's clear to other maintainers that this is the same endpoint as in v1.